### PR TITLE
autoclick

### PR DESCRIPTION
**Pull request by alinpr18**
Add new extension for the G-ExtensionStore

### CODE REVIEW

File: store/extensions/autoclick/extension.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "title": "autoclick",
+  "description": "Automatic clicks in any position in the room.",
```
Comment (WiredSpast) [inline store/extensions/autoclick/extension.json]: Maybe include the `:autoclick` command in the description

File: store/extensions/autoclick/extension.zip
Comment (WiredSpast) [inline store/extensions/autoclick/extension.zip:1]: Make sure the extensions main files are directly in the base of the zip
image

You're also missing the NodeExtensionSetup.jar file in the zip, you can find that here: https://github.com/WiredSpast/NodeExtensionSetup/releases/latest

File: store/extensions/autoclick/extension.json (hunk 1 above)
Comment (alinpr18) [reply store/extensions/autoclick/extension.json]: Ready, I'm done

Comment (WiredSpast) [reply store/extensions/autoclick/extension.json]: > Ready, I'm done

You still have to fix the zip, as listed below

**Pull request merged**